executor/instruction/iterator.rs

```diff
@@ -50,7 +50,7 @@ use crate::{
         relates_executor::{RelatesBoundedSortedRole, RelatesUnboundedSortedRelation},
         relates_reverse_executor::{RelatesReverseBoundedSortedRelation, RelatesReverseUnboundedSortedRole},
         sub_executor::{SubBoundedSortedSuper, SubUnboundedSortedSub},
-        sub_reverse_executor::{SubReverseBoundedSortedSuper, SubReverseUnboundedSortedSub},
+        sub_reverse_executor::{SubReverseBoundedSortedSub, SubReverseUnboundedSortedSuper},
```

Comment:
Looked like a typo based on the actual usage

Reply:
I think I didn't update a lot of these when I adjusted tuple ordering for these iterators. Note to self to fix in my next PR (out of scope for this one).
Good catch!